docs/source/config.rst

```diff
@@ -11,47 +11,47 @@ File format
+- ``"header"``: list(str) queuing system header lines.  Actual header is created by applying string formatting, i.e.  ``str.format(**node_dict)``, replacing substrings such as ``{nnodes}``.
 - ``rundir``: str, optional, default ``"run_expyre"``. Place for all remote files and remote execution. 
-- ``no_default_header``: bool, optional, default False``. Do not put default lines (such as ``#$ -cwd`` or automatically setting the job name) to the job submission script. 
+- ``no_default_header``: bool, optional, default ``False``. Do not put default lines (such as ``#$ -cwd`` or automatically setting the job name) to the job submission script. 
+- ``"remsh_cmd"``: optional string remote shell command, default ``"ssh"``
+- ``"no_default_header"``: bool, default false disable automatic setting of queuing system header for job name, partition/queue, max runtime, and stdout and stderr files
+- ``"rundir"``: string for a path where the remote jobs should be run
+- ``"partitions"``: dict with partitions/queues/node-type names as keys and dict of node properties as values.
+
+===========================
+Node property dict includes
+===========================
+
+- ``"ncores"``: int number of cores per node
+- ``"max_time"``: max time, int for seconds, str for ``"<N>[smhd]"`` (case insensitive) or ``"<dd>-<hh>:<mm>:<ss>"``. Leading parts are optional, so N1:N2 is N1 minutes + N2 seconds.
+- ``"max_mem"``: max mem per node, int for kB, str for ``"<N>[kmgt]b?"`` (case insensitive).
 
-Available keys in ``kwargs`` which are normally used in this template:
+==========================
+format() keys used in header
+==========================
+
+Available keys in ``node_dict`` which are normally used in this template:
 
 - ``"nnodes"``: int total number of nodes
+- ``"ncores"``: int total number of cores
 - ``"ncores_per_node"``: int number of cores per node
-- ``"tot_ncores"``: int total number of cores
 
 Note that if ``partial_node=True`` is passed to ``find_nodes`` and the total number of cores is less
-than the number per node, ``tot_ncores`` and ``ncores_per_node`` are *not* rounded up to an entire node.
-
-Additional keys that are not normally used for the header (having to do with two level parallelism where each task may be allocated more than 1 node):
-
-- ``"tot_ntasks"``: int total number of tasks (``tot_ncores / ncores_per_task`` or ``nnodes * ntasks_per_node``)
-- ``"ncores_per_task"``: int cores per task (specified in resources)
-- ``"ntasks_per_node"``: int tasks per node (``ncores_per_node / ncores_per_task``)
+than the number per node, ``ncores`` and ``ncores_per_node`` are *not* rounded up to an entire node.
 
 Additional keys that are used by the internally generated parts of the header:
 
 - ``"id"``: str (supposed to be guaranteed to be unique among current jobs within project) job id
 - ``"partition"``: str partition/queue/node type
```

Comment:
`queue` works as well?

Reply:
I'll make sure it does.

Reply:
Added it as a synonym when `node_dict` is created.